### PR TITLE
Just a comment on #2983

### PR DESCRIPTION
**Pull request by erights**
Just a comment on #2983

### CODE REVIEW

**Review by katelynsills — APPROVED** [review]
Looks good to me! I had copied these from `master` without assessing them, but this is a nice improvement!

**Comment by katelynsills** [comment]
Ah, wait, this changed the error message to `payment not found for "Zoe Invitation`

File: packages/zoe/src/zoeService/invitationQueries.js [hunk 1]
```diff
@@ -4,36 +4,33 @@ import { assert, details as X } from '@agoric/assert';
...
-    return E(invitationIssuer)
-      .getAmountOf(invitationP)
-      .then(onFulfilled, onRejected);
+  /** @type {GetInvitationDetails} */
+  const getInvitationDetails = async invitationP => {
+    const onRejected = reason => {
+      // Or you could do it the other way around
+      assert.note(reason, X`A Zoe invitation is required, not ${invitationP}`);
+      throw reason;
+    };
```
Comment (erights) [inline packages/zoe/src/zoeService/invitationQueries.js]: The other way around should keep the error you want at the top, while still linking to the original error
```suggestion
    const onRejected = reason => {
      const err = assert.error(X`A Zoe invitation is required, not ${invitationP}`);
      assert.note(err, X`Due to ${reason}`);
      throw err;
    };
```

**Review by erights — COMMENTED** [review]
@katelynsills see my suggested change

**Pull request merged**